app/models/provider.rb

```diff
@@ -130,23 +130,23 @@ def current_accredited_courses
 
   validates :email, email_address: true, if: :email_changed?
 
-  validates :provider_name, length: { maximum: 100 }, on: :update
+  validates :provider_name, presence: true, length: { maximum: 100 }
```

Comment:
Wondering why we're dropping the `on: :update`

Reply:
We are creating a new provider, I think?